src/lib/utils/interfaces/team.interface.ts

```diff
@@ -43,4 +43,5 @@ export interface TeamServiceMethods {
     body: ChangeRoleBody,
   ) => unknown;
   removeUserFromWorkspace?: (workspaceId: string, userId: string) => unknown;
+  disableNewInviteTag?: (userId: string, teamId: string) => unknown;
```

Comment:
makeRequest either returns a error or a success json as defined in api.common.ts. Can we make a type and use it here instead of unknown.

example - RequestSuccess | RequestError

Reply:
We can use the same type in other methods also that is calling makeRequest.

Reply:
resolved